accounts/tests.py

```diff
@@ -2,6 +2,8 @@
 from django.test import TestCase
 from django.urls import reverse
 
+from mysite import settings

```

Comment:
settingsモジュールはdjango.confからインポートするようにしましょう。
理由は単純でmysiteにsettingsファイルが置いてあるとは限らないからです。
```suggestion
from django.conf import settings
```

Reply:
mysiteからdjango.confに変更しました！[5db26b5](https://github.com/shinonome-inc/backend-final-assignment-Koshi/pull/2/commits/5db26b596c8a48f4bf5bd86b686ee23d0c57a1f3)